test/test.go

```diff
@@ -0,0 +1,141 @@
+	tx = append(tx, bin...)
+	tx = append(tx, uint64_to_buf(1000000)...) // gas limit
+	tx = append(tx, uint64_to_buf(1000)...) // gas price
+	tx = append(tx, uint64_to_buf(1000000000000)...) // amount
+	// call data
+	bin, _ = hex.DecodeString("00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000") // bin data
+	tx = append(tx, bin...)
+	bin, _ = hex.DecodeString("00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000") // bin data
+	tx = append(tx, bin...)
+	bin, _ = hex.DecodeString("00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000") // bin data
+	tx = append(tx, bin...)
+	bin, _ = hex.DecodeString("00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000") // bin data
+	tx = append(tx, bin...)
+	tx = append(tx, publicKey.PublicKey...)
+
+	response, err = device.SignTx(ledger.StringToPath("44'/540'/0'/0/0'"), tx)
+	if err != nil {
+		fmt.Printf("Verify spawn tx ERROR: %v\n", err)
+	} else {
+		hash := sha512.Sum512(tx)
+		fmt.Printf("Verify spawn tx: %v\n", ed25519.Verify(publicKey.PublicKey, hash[:], response[1:65]))
+	}
+}
+
+func main() {
+	devices := ledger.GetDevices(0)
```

Comment:
If no device found it's hard to tell that the test didn't do anything (one could assume it passed).
```suggestion
	devices := ledger.GetDevices(0)
	if len(devices) == 0 {
		fmt.Printf("ERROR: no devices found\n")
	}
```